runbotics-orchestrator/src/main/java/com/runbotics/domain/BotSystem.java

```diff
@@ -13,6 +15,7 @@ public class BotSystem {
     @Size(max = 50)
     @Id
     @Column(length = 50)
+    @JsonView(ProcessDTOViews.DefaultView.class)
```

Comment:
I'm not sure what this line of code is here for. ProcessDTO contains BotSystem system field, which is already marked in this way